lib/beaker-hostgenerator/parser.rb

```diff
@@ -110,14 +113,22 @@ def tokenize_layout(layout_spec)
       # due to its unlikely occurrence in the user input string.
       spec = String.new(layout_spec) # Copy so we can replace characters inline
       within_braces = false
+      within_brackets = false
       spec.chars.each_with_index do |char, index|
         case char
         when '{'
           within_braces = true
         when '}'
           within_braces = false
+        when '['
+          within_brackets = true
+        when ']'
+          within_brackets = false
         when '-'
           spec[index] = '|' if within_braces
+        # replacing commas in arbirary list sections
+        when ','
+          spec[index] = '~' if within_brackets
```

Comment:
It would only affect things inside of a list as the replace in `settings_string_to_map` doesn't look at anything outside the `[]`. But yeah for `home=[~/foo,bar]` is would break and split improperly. What symbol would suggest instead?